### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by TedLivist**
In this project, I:
- Wrapped `App` component in `BrowserRouter` Component from `react-router-dom`.
- Added routing for `Home`, `Calculator` and `Quote` components in `App` component.
- Added and styled `Navbar` component.
- Made `Navbar` links work.
- Added and styled `Home`, `Calculator` and `Quote` components.

### CODE REVIEW

File: src/pages/quote.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable arrow-body-style, react/jsx-one-expression-per-line */
```
Comment (arslanbisharat) [inline src/pages/quote.js]: - [ ] Let's not disable eslint rules. These are here to help you write best code ever.

File: src/pages/home.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/* eslint-disable arrow-body-style, react/jsx-one-expression-per-line */
```
Comment (arslanbisharat) [inline src/pages/home.js]: - [ ] Let's not disable eslint rules. These are here to help you write best code ever.

File: src/components/navbar.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable arrow-body-style */
```
Comment (arslanbisharat) [inline src/components/navbar.js]: - [ ] Let's not disable eslint rules. These are here to help you write best code ever.

File: README.md [hunk 4]
```diff
@@ -4,6 +4,8 @@
 
 > "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
 
+![screenshot](./src/images/scrnsht.jpg)
+
```
Comment (arslanbisharat) [inline README.md:8]: - [ ] Let's add a live demo link to your project so anyone can access it easily.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @TedLivist, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**